src/bindings/g_object/binding.yml

```diff
@@ -79,3 +79,7 @@ ignore:
  - ParamSpecVariant
  - SignalQuery
  - WeakRef
+
+execute_callback:
+ - g_signal_emit_by_name
```

Comment:
I just noticed that this function is not on GObjectInstrospection, it's added manually by the generator itself at `lib.ecr` at line 42, so the annotation must be put there and we don't need this entry.

Sorry for the confusion.